lib/helpers/global_helper.js

```diff
@@ -15,6 +15,10 @@ if (Meteor.isClient) {
 		return Meteor.userId();
 	});
 
+	Template.registerHelper('name', function () {
+		return Meteor.users.find({_id: Session.get('offerSelected').creator_id}).fetch()[0].profile.name;
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Meteor' is not defined.<br>'Session' is not defined.